examples/intro_to_observations.jl

```diff
@@ -0,0 +1,146 @@
+# # Intro to observations
+#
+# This example demonstrates the features of `OneDimensionalTimeSeries`
+# when constructed from  "synthetic observations" generated by an Oceananigans `Simulation`.
+#
+# ## Install dependencies
+#
+# First let's make sure we have all required packages installed.
+
+# ```julia
+# using Pkg
+# pkg"add OceanTurbulenceParameterEstimation, Oceananigans, CairoMakie"
+# ```
+
+# First we load few things
+
+using OceanTurbulenceParameterEstimation
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.TurbulenceClosures: ConvectiveAdjustmentVerticalDiffusivity
+using CairoMakie
+
+# # Generating synthetic observations
+#
+# We define a utility function for constructing synthetic observations,
+
+function generate_free_convection_synthetic_observations(name = "convective_adjustment";
+                                                         Nz = 32,
+                                                         Lz = 64,
+                                                         Qᵇ = +1e-8,
+                                                         Qᵘ = -1e-5,
+                                                         Δt = 10.0,
+                                                         f₀ = 1e-4,
+                                                         N² = 1e-6)
+    data_path = name * ".jld2"
+
+    if isfile(data_path)
+        return data_path
+    end
+
```

Comment:
What's this for?

Reply:
if the data already exists, don't run the simulation. it makes the docs build less expensive, because if the file was created by a previous example, it won't be re-created.

Reply:
Oh I see!!
`return` forces to not proceed with whatever else is in the function! I forgot that :)